dhcpv4/dhcpv4.go

```diff
@@ -190,74 +185,46 @@ func NewInformForInterface(ifname string, needsBroadcast bool) (*DHCPv4, error)
 	return pkt, nil
 }
 
-// NewInform builds a new DHCPv4 Informational message with default Ethernet HW
-// type and specified hardware address. It does NOT put a DHCP End option at the
-// end.
-func NewInform(hwaddr net.HardwareAddr, localIP net.IP) (*DHCPv4, error) {
-	d, err := New()
-	if err != nil {
-		return nil, err
-	}
+// PrependModifiers prepends other to m.
+func PrependModifiers(m []Modifier, other ...Modifier) []Modifier {
```

Comment:
is this really necessary? It's basically a synonym of `append`, can't we just do `New(append(modifiers, WithHwAddr(..), ... )` ?

Reply:
For one, it's intentionally the other way (prepend, instead of append), so that if you do 

NewReplyForRequest(..., WithSomething())

the WithSomething can override options that NewReplyForRequest enacts.

For example,

NewReplyForRequest(ack, WithHwAddr(somethingelse))

in the prepend world, somethingelse becomes the ClientHWAddr. in the append world, ack.ClientHWAddr becomes the ClientHWAddr.

Reply:
sorry, I didn't read it right, makes sense now